layouts/v2-sidebar.html

```diff
@@ -17,6 +17,7 @@ <h4 class="sidebar-title">API v2</h4>
   <li><ul class="nav-list">
     <li><a href="/v2/domains/certificates/">Certificates</a></li>
     <li><a href="/v2/domains/email-forwards/">Email Forwards</a></li>
+    <li><a href="/v2/domains/templates/">Templates</a></li>
```

Comment:
[There is](https://github.com/dnsimple/dnsimple-developer/blob/master/layouts/v2-sidebar.html#L37) already a Template link in the sidebar.